executor/runtime/docker/docker.go

```diff
@@ -1830,6 +1830,9 @@ func (r *DockerRuntime) k8sContainerToDockerConfigs(v1Container v1.Container, ma
 		VolumesFrom: []string{mainContainerID},
 		Mounts:      mounts,
 		Init:        &b,
+		Tmpfs: map[string]string{
+			"/run": "rw,exec,size=" + defaultRunTmpFsSize,
```

Comment:
should this be `/run` or `/tmp`?

Reply:
`/run`, to match existing behavior with the main container.

Titus doesn't seem to do anything special with `/tmp` and assumes images come with it.

Reply:
Would this be a /run (of that size) per container? Shouldn't this be shared?